cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -172,13 +174,21 @@ func (command *Evaluate) Execute(args []string) (err error) {
 		}
 	}
 
+	// Combine multiple repository assessments for reporting.
+	assessmentsPerModel := make(map[string]metrics.Assessments, len(assessmentsPerModelPerRepository))
```

Comment:
Can you print assessmentsPerModelPerRepository before to the CSV and then do the logic for assessmentsPerModel in one block to print the scores.

Reply:
I think that is better grouping: print overall results first and then combine into different reports.